<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgQComboBox_QTableView</class>
 <widget class="QDialog" name="dlgQComboBox_QTableView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo del Control QComboBox y QTableView</string>
  </property>
  <widget class="QLabel" name="lblData">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>511</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Selecciona un elemento de la lista</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboData">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tblData">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1181</width>
     <height>731</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
